docs/setup/kubeedge_cri_configure.md

```diff
@@ -65,7 +65,7 @@ CONTAINER                                                           IMAGE
 41c1a07fe7bf7425094a9b3be285c312127961c158f30fc308fd6a3b7376eab2    docker.io/library/nginx:1.15.12    io.containerd.runtime.v1.linux
 ```
 
-*NOTE: namespace here might be changed soon from “k8s.io” to “kubeedge.io”.*
+NOTE: since cri doesn't supports multi-tenancy while containerd does, namespace for containers are set to "k8s.io" defaultly and no way to change that until the [cri's support](https://github.com/containerd/cri/pull/1462) has done.
```

Comment:
With containerd,  containers  will use the namespace from k8s/cri?

Reply:
I think so, you can see the source from cri, https://github.com/containerd/cri/blob/7361cf86217573819ad210891a552060275730a6/cri.go#L91-L95

`K8sContainerdNamespace` is defined as "k8s.io"

Reply:
👍  So containerd use a default ns like `K8sContainerdNamespace` instead of getting it from pod's namespace? :)  cc @kadisi  
And do users need to care the namespace config for containerd/cri-o when they use k8s/kubeedge?

Reply:
Ohh, users can change `k8s.io` to `kubeedge.io` if they need.

Reply:
this should be documented as this, user interesting in the multi-tenancy might need this info, afaik, k3s is extending their capability on the multi-tenacy.

Reply:
> Ohh, users can change `k8s.io` to `kubeedge.io` if they need.

this cannot be changed by far, regardless which ns is specified in the pod, the containers will be created under "k8s.io" instead on edge side.

Reply:
but for k8s, cli doesn't aware all those things. k8s cli on cloud will show the pod as usual but the containers is created under "k8s.io" on edge side.